lib/core/src/lib/card-view/components/card-view-dateitem/card-view-dateitem.component.ts

```diff
@@ -57,36 +53,38 @@ export class CardViewDateItemComponent extends BaseCardView<CardViewDateItemMode
     @Input()
     displayClearAction: boolean = true;
 
-    @ViewChild('datetimePicker')
-    public datepicker: MatDatetimepickerComponent<any>;
+    @ViewChild('datePicker')
+    public datepicker: MatDatepicker<any>;
 
-    valueDate: Moment;
+    valueDate: Date;
     dateFormat: string;
 
     private onDestroy$ = new Subject<boolean>();
 
-    constructor(private dateAdapter: DateAdapter<Moment>,
+    constructor(private dateAdapter: DateAdapter<DateFnsAdapter>,
                 private userPreferencesService: UserPreferencesService,
-                private appConfig: AppConfigService,
                 private clipboardService: ClipboardService,
-                private translateService: TranslationService) {
+                private translateService: TranslationService,
+                @Inject(MAT_DATE_FORMATS) private dateFormatConfig: MatDateFormats) {
```

Comment:
never ever do this, this is a singleton access to all the date time formats, changing it will change all components